pump/omnipod-common/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/common/ui/wizard/deactivation/PodDeactivationWizardActivity.kt

```diff
@@ -1,13 +1,15 @@
 package info.nightscout.androidaps.plugins.pump.omnipod.common.ui.wizard.deactivation
 
 import android.os.Bundle
+import android.view.WindowManager
 import info.nightscout.androidaps.plugins.pump.omnipod.common.R
 import info.nightscout.androidaps.plugins.pump.omnipod.common.ui.wizard.common.activity.OmnipodWizardActivityBase
 
 abstract class PodDeactivationWizardActivity : OmnipodWizardActivityBase() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
```

Comment:
Should not be needed here as this class inherent from OmnipodWizardActivityBase in which you made the same change

Reply:
fixed